src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
```

Comment:
don't add '_' to blocks file name, it's only for utils files

Reply:
pay attention to comments, no need to use `_` in block files names